tests/integration/customer/test_auth_backend.py

```diff
@@ -0,0 +1,21 @@
+import django
+from django.test import TestCase
+
+from oscar.apps.customer.auth_backends import EmailBackend
+from oscar.test.factories import UserFactory
+
+
+class AuthBackendTestCase(TestCase):
+
+    def test_authentication_method_signature(self):
+        user = UserFactory(email='foo@example.com', is_staff=True)
+        user.set_password('letmein')
+        user.save()
+
+        backend = EmailBackend()
+        if django.VERSION < (1, 11):
+            auth_result = backend.authenticate('foo@example.com', 'letmein')
+        else:
+            auth_result = backend.authenticate(None, 'foo@example.com', 'letmein')
```

Comment:
Maybe we can just split it into two conditional tests using `unittest.skipUnless`? That way we can make sure that test for particular Django version passed or failed.